main.js

```diff
@@ -83,6 +83,15 @@ Trello.prototype.addListToBoard = function (boardId, name, callback) {
     return makeRequest(rest.post, this.uri + '/1/boards/' + boardId + '/lists', {query: query}, callback);
 };
 
+Trello.prototype.addMemberToBoard = function (boardId, memberId, type, callback) {
+    var query = this.createQuery();
+    var data = {};
```

Comment:
If all we do with `data` is put `type` in it, then why not shorten

```javascript
var data = {}
data.type = type
```

to

```javascript
var data = {
  type: type
}
```

?